wolfssl/ssl.h

```diff
@@ -3346,7 +3347,9 @@ WOLFSSL_API void wolfSSL_sk_X509_NAME_pop_free(WOLF_STACK_OF(WOLFSSL_X509_NAME)*
     void f (WOLFSSL_X509_NAME*));
 WOLFSSL_API void wolfSSL_sk_X509_NAME_free(WOLF_STACK_OF(WOLFSSL_X509_NAME) *);
 
-
+#if defined(WOLFSSL_APACHE_HTTPD)
+WOLFSSL_API int get_dn_attr_by_nid(int n, char** buf);
```

Comment:
Should this be a public API? Guessing you can remove from header and make the function `static`.

Reply:
Ah, you're correct. I guess I added it out of habit!